cmd/helm/installer/init.go

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package installer // import "k8s.io/helm/cmd/helm/installer"
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"k8s.io/helm/pkg/getter"
+	helm_env "k8s.io/helm/pkg/helm/environment"
+	"k8s.io/helm/pkg/helm/helmpath"
+	"k8s.io/helm/pkg/repo"
+)
+
+const (
+	stableRepository = "stable"
+
+	// LocalRepository is the standard name of the local repository
+	LocalRepository = "local"
+
+	// LocalRepositoryIndexFile is the standard name of the local repository index file
+	LocalRepositoryIndexFile = "index.yaml"
+)
+
+// Initialize initializes local config
+//
+// Returns an error if the command failed.
+func Initialize(home helmpath.Home, out io.Writer, skipRefresh bool, settings helm_env.EnvSettings, stableRepositoryURL, localRepositoryURL string) error {
```

Comment:
Maybe this should be named `EnsureHome()`?

Reply:
Yeah, EnsureHome would be a bit more clearer on intent.